### ISSUE TITLE
blowing up on NYC's audit

### ISSUE THREAD

**Issue by patwater** [tabulapdf/tabula#270]
Tried several tables within this and getting error messages each time: http://comptroller.nyc.gov/wp-content/uploads/documents/CAFR2014.pdf

**Comment by jeremybmerrill** [comment]
Hi Patrick,

Funny running into you here... (CMC '12 and one of the Tabula developers) 

I checked out a few tables from the report and they worked fine for me. Can you make sure you have the [latest version of Tabula](https://github.com/tabulapdf/tabula/releases/tag/v0.9.7) (there was a new minor release a few weeks ago)? If so, can you include the exact error message you received?

**Comment by patwater** [comment]
Jeremey,

Ha small world.  Here's the error though it only seems to be happening on
the first few quasi-lists / tables.  Btw do you know anyone who's
integrated tabula into a pdf extraction pipeline?  i.e. programatically
parsing similar pdfs as achron job

Interning at enigma.io and playing aroudn with something in that vein

Sorry, there's been an error

*Tabula* couldn't finish processing your request. We'd appreciate if you
could report this error in our issue tracker
<https://github.com/jazzido/tabula/issues/new>. If possible, please include
the document that caused the error and the text below.

On Thu, Feb 19, 2015 at 4:40 PM, Jeremy B. Merrill <notifications@github.com
> wrote:

> Hi Patrick,
>
> Funny running into you here... (CMC '12 and one of the Tabula developers)
>
> I checked out a few tables from the report and they worked fine for me.
> Can you make sure you have the latest version of Tabula
> <https://github.com/tabulapdf/tabula/releases/tag/v0.9.7> (there was a
> new minor release a few weeks ago)? If so, can you include the exact error
> message you received?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tabulapdf/tabula/issues/270#issuecomment-75141874>.
>



-- 
Patrick Atwater | 818.259.2709 | patrickatwater@gmail.com
[A]dvanced [R]esearch in [G]overnment [O]perations
<http://argomarketplace.com>

**Comment by jeremybmerrill** [comment]
Yes, I do. It's mostly me though. Check out [tabula-extractor](https://github.com/tabulapdf/tabula-extractor) -- it's a command line tool that can also be used as a Ruby gem. 

Public examples include this NYPD compstat scraper I just open-sourced (but haven't announced yet): https://github.com/nytinteractive/compstat_parser . There may be a few others out there.

Your colleague (and my former colleague, small world, yeah) [put in a note](https://github.com/tabulapdf/tabula/issues/251) a few weeks ago asking about API access, for if you're in Python.

**Comment by jeremybmerrill** [comment]
And I still can't replicate the error :-/

**Comment by patwater** [comment]
Oh awesome stuff.  That compstat tool is pretty epic.  Do you mind if I
share it with a couple folks at CUSP?  Cheers

On Thu, Feb 19, 2015 at 5:07 PM, Jeremy B. Merrill <notifications@github.com
> wrote:

> And I still can't replicate the error :-/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tabulapdf/tabula/issues/270#issuecomment-75148693>.
>



-- 
Patrick Atwater | 818.259.2709 | patrickatwater@gmail.com
[A]dvanced [R]esearch in [G]overnment [O]perations
<http://argomarketplace.com>

**Comment by jeremybmerrill** [comment]
totally, go fo rit
